fastlane/lanes/localization.rb

```diff
@@ -243,11 +243,7 @@ def generate_strings_file(gutenberg_path:, derived_data_path:)
 
     push_to_git_remote(tags: false)
 
-    pr_url = create_release_management_pull_request(
-      release_version: release_version,
-      base_branch: compute_release_branch_name(options: options, version: release_version),
-      title: "Merge editorialized release notes in #{release_version}"
-    )
+    pr_url = create_backmerge_pr
```

Comment:
@iangmaia @mokagio Unfortunately this change is not correct. Please see [this comment from WPAndroid](https://github.com/wordpress-mobile/WordPress-Android/pull/21229/files#r1800099465) for the same issue.

Reply:
For the record, this PR should fix this issue: https://github.com/wordpress-mobile/WordPress-iOS/pull/23758